src/NpmUtilities.js

```diff
@@ -76,21 +76,21 @@ export default class NpmUtilities {
   }
 
   @logger.logifySync()
-  static addDistTag(packageName, version, tag, registry) {
+  static addDistTag(directory, packageName, version, tag, registry) {
```

Comment:
I changed `getTagOpts` signature to accept directory in addition to registry. `publishTaggedInDir` was already using the same `cd ... && npm ...` method. I updated this so that all methods are now using your suggestion